Use custom message feedback

printminion-co opened the pull request.

printminion-co committed:
feat(message-feedback): allow feedback without rating

in order to reuse this functionality later

Signed-off-by: Misha M.-Kupriyanov <kupriyanov@strato.de>

printminion-co committed:
feat(message-feedback): trigger feedback dialog via store value

in order to be able to trigger feedback globally

Signed-off-by: Misha M.-Kupriyanov <kupriyanov@strato.de>

printminion-co committed:
feat(message-feedback): switch to alternative feedback dialog on messages

Signed-off-by: Misha M.-Kupriyanov <kupriyanov@strato.de>

bromiesTM commented on src/lib/stores/index.ts:
```diff
@@ -52,6 +52,13 @@ export const temporaryChatEnabled = writable(false);
+
+type FeedbackDialog = {
+	open: boolean;
+	rating?: number
```
suggestion: rating is now `number | undefined` but should he `numbern | null`

```suggestion
	rating: number | null
```

bromiesTM replied:
and prettier says
```suggestion
	rating?: number;
};
````

bromiesTM committed:
fix: FEEDBACK_BASE_URL not set at build time
Using a dynamic environment variable import to prevent errors during build time as the variable is optional and implemented for local development environments

Signed-off-by: Kai Henseler <kai.henseler@strato.de>

bromiesTM commented on src/lib/components/chat/Messages/ResponseMessage.svelte:
```diff
@@ -344,6 +346,15 @@
+			feedbackLoading = false;
+			feedbackDialog.set({
+				open: true,
+				rating,
```
prettier says 
```suggestion
				rating
```
